src/shared/services/WalletService/WalletService.ts

```diff
@@ -19,10 +19,11 @@ function createWalletService(mirrorNodeService: MirrorNodeServiceType) {
     return hashconnect.getSigner(AccountId.fromString(accountId));
   };
 
-  const reconnectToOtherNetwork = async (network: LedgerId) => {
-    await disconnect();
-    localStorage.setItem("activeNetwork", network?.toString());
-    setTimeout(() => window.location.reload(), 1000);
+  const reconnectToOtherNetwork = (network: LedgerId) => {
+    disconnect().then(() => {
+      localStorage.setItem("activeNetwork", network.toString());
+      setTimeout(() => window.location.reload(), 2000);
```

Comment:
Why it was needed to increase the timeout? How can we make sure that now it's enough?

Reply:
> Why it was needed to increase the timeout? How can we make sure that now it's enough?

I tested this stuff few and more times. Actually the main point is that put `await disconnect()` should keep waiting for disconnect to happen. So it means that it should work without `setTimeout()` at all, but @web3-nomad found related issue during domain deployment and it means we need something like `setTimeout()`.

Reply:
@maryjanyes I'd suggest to add a comment with a description of the issue